octoprint/Dockerfile.template

```diff
@@ -1,12 +1,16 @@
-FROM --platform=linux/arm/v7 nunofgs/octoprint:master-alpine
```

Comment:
@chrisys I didn't get why this was forced to a specific platform before. It seemed to work without it with the new image...